src/router/router.tsx

```diff
@@ -18,35 +17,24 @@ export const Router = () => {
   );
 
   return (
-    <>
-      <Routes location={previousLocation ?? location}>
-        {/* PUBLIC ONLY ROUTES */}
-        {userState === "loggedOut" && (
-          <>
-            <Route
-              element={<Navigate to={ROUTES.login} replace />}
-              path={"*"}
-            />
-            <Route element={<Login />} path={ROUTES.login} />
-          </>
-        )}
-        {/* PRIVATE ONLY ROUTES */}
-        {userState === "loggedIn" && (
-          <>
-            <Route element={<Navigate to={ROUTES.home} />} path={"*"} />
-            <Route path="*" element={<NotFound />} />
-            <Route element={<Layout />}>
-              <Route element={<Home />} path={ROUTES.home} />
-            </Route>
-          </>
-        )}
-      </Routes>
-      <Routes>
-        <Route
-          path="*"
-          element={<ModalRouter showModal={!!previousLocation} />}
-        />
-      </Routes>
-    </>
+    <Routes location={previousLocation ?? location}>
+      {/* PUBLIC ONLY ROUTES */}s
+      {userState === "loggedOut" && (
+        <>
+          <Route element={<Navigate to={ROUTES.login} replace />} path={"*"} />
+          <Route element={<Login />} path={ROUTES.login} />
+        </>
+      )}
+      {/* PRIVATE ONLY ROUTES */}
+      {userState === "loggedIn" && (
+        <>
+          <Route element={<Navigate to={ROUTES.home} />} path={"*"} />
+          <Route path="*" element={<NotFound />} />
```

Comment:
I believe you are never going to NotFound path because you are redirected to Home first.

Reply:
Yeah, I just stripped the Modal Router, good catch 👍